pkg/looker/resource_user_attribute.go

```diff
@@ -76,7 +77,12 @@ func resourceUserAttributeCreate(ctx context.Context, d *schema.ResourceData, m
 		ValueIsHidden:              &userAttributeValueIsHidden,
 		UserCanView:                &userAttributeUserCanView,
 		UserCanEdit:                &userAttributeUserCanEdit,
-		HiddenValueDomainWhitelist: &userAttributeHiddenValueDomainWhitelist,
+	}
+
+	if userAttributeValueIsHidden && userAttributeHiddenValueDomainWhitelist != "" {
+
+		writeUserAttribute.HiddenValueDomainWhitelist = &userAttributeHiddenValueDomainWhitelist
+
 	}
```

Comment:
Could you remove redundant empty lines like below?

```suggestion
	if userAttributeValueIsHidden && userAttributeHiddenValueDomainWhitelist != "" {
		writeUserAttribute.HiddenValueDomainWhitelist = &userAttributeHiddenValueDomainWhitelist
	}
```

Reply:
Is this just go formatting?

Reply:
oh i see just new lines - :+1: